test/suite/test_tiered19.py

```diff
@@ -26,10 +26,14 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import wttest
+import wttest, wiredtiger, random, string
```

Comment:
Turbo nitpick, it's conventional to alphabetise these (it makes it easier for readers to see where a package came from).

Reply:
fixed